src/mutations/inviteShopMember.js

```diff
@@ -45,8 +46,19 @@ export default async function inviteShopMember(context, input) {
   const shop = await Shops.findOne({ _id: shopId });
   if (!shop) throw new ReactionError("not-found", "No shop found");
 
-  const group = await Groups.findOne({ _id: groupId });
-  if (!group) throw new ReactionError("not-found", "No group found");
+  const groups = await Groups.find({
+    _id: {
+      $in: groupIds
+    }
+  }).toArray();
+
+  if (!groups) {
```

Comment:
Will never be true since `.toArray()` guarantees an array returned. You'd need to check for length `0`